src/test/java/uk/gov/companieshouse/mapper/OtherReasonMapperTest.java

```diff
@@ -10,15 +10,15 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.companieshouse.TestData;
 import uk.gov.companieshouse.database.entity.AttachmentEntity;
 import uk.gov.companieshouse.database.entity.OtherReasonEntity;
 import uk.gov.companieshouse.model.Attachment;
 import uk.gov.companieshouse.model.OtherReason;
 import uk.gov.companieshouse.TestUtil;
 
-@ExtendWith(SpringExtension.class)
+@ExtendWith(MockitoExtension.class)
 public class OtherReasonMapperTest {
```

Comment:
Remove **public** from class declaration. Also, we use the **@DisplayName** annotation on test methods to give a more verbose way of describing a test. Can this annotation be added to test methods in the class please.

Reply:
Done in new commit